plugins/aggregators/starlark/testdata/merge.star

```diff
@@ -0,0 +1,25 @@
+# Example of a merge aggregator implemented with a starlark script.
+
+load('time.star', 'time')
+
+def add(cache, metric):
+    metrics = cache.get("metrics")
+    if metrics == None:
+        metrics = {}
+        cache["metrics"] = metrics
+        cache["ordered"] = []
+    m = metrics.get(metric)
```

Comment:
I don't understand using metric as a key name here. is it automatically converted to a string or something?

Reply:
It is possible because `metrics` is a dictionary which is actually [an hashtable](https://github.com/google/starlark-go/blob/master/starlark/hashtable.go) and I added [an implementation of the function `Hash()`](https://github.com/influxdata/telegraf/pull/9419/files#diff-68dab7d4d08265bd0c597d5e5106b8b9fc5d15ea3ebe48743c3a8f7ef9b6f172R64) to be able to store a metric as a key